components/Map/index.js

```diff
@@ -0,0 +1,79 @@
+import { MapContainer, Marker, Popup, TileLayer } from "react-leaflet";
+import styled from "styled-components";
+import * as L from "leaflet";
+import "leaflet/dist/leaflet.css";
+import "leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.css";
+import "leaflet-defaulticon-compatibility";
+import CurrentOutivityMarker from "./CurrentOutivityMarker";
+import Link from "next/link";
+
+export default function Map({ currentOutivity = {}, outivities, outivity }) {
+  return (
+    <StyledMapContainer
+      center={[outivity.lat, outivity.long]}
+      zoom={10}
+      scrollWheelZoom
+    >
+      <TileLayer url="https://tiles.stadiamaps.com/tiles/alidade_smooth/{z}/{x}/{y}{r}.png" />
+
+      {outivities.map((outivity) =>
+        outivity.id === currentOutivity.id ? (
+          <CurrentOutivityMarker
+            key={outivity.id}
+            outivity={outivity}
+            position={[outivity.lat, outivity.long]}
+          />
+        ) : (
+          <Marker
+            key={outivity.id}
+            position={[outivity.lat, outivity.long]}
+            icon={greyIcon}
```

Comment:
As far as I see it, the only reason you have the CurrentOutivityMarker now is, that the icon-color is blue, not grey, right? The center of the map is set in line 13 to the slug-outivity, that you give the Map function as a prop, anyway. So I think it would be a much cleaner code if you just made the type of icon conditional to the currentOutivity.id. Then you can delete the entire CurrentOutivityMaker.js, right? 
The condition should be something like this: (I'm not sure about the correct syntax right now - it's late, I shouldn't review code right now 😅
```suggestion
            icon={outivity.id === currentOutivity.id ? blueIcon : greyIcon}
```